docs/core/migration/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: .NET Core project.json to csproj migration | Microsoft Docs
+description: .NET Core project.json to csproj migration
+keywords: .NET, .NET Core, .NET Core migration
+author: blackdwarf
+ms.author: mairaw
+ms.date: 02/17/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 1feadf3d-3cfc-41dd-abb5-a4fc303a7b53
+---
+
+# Migrating .NET Core projects
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+This document will cover migration scenarios for .NET Core projects and will go over the following three migration scenarios:
+
+1. Migration from a valid latest schema of *project.json* to *csproj*.
+2. Migration from DNX to a valid Preview 2 project.json.
+3. Migration from RC3 and previous .NET Core csproj projects to the final format .
+
+## Migration from project.json to csproj
+Migration from project.json to csproj can be done via the [`dotnet migrate`](../preview3/tools/dotnet-migrate.md) command-line tool or through Visual Studio 2017. The command-line tool and Visual Studio 2017 use the same underlying engine to migrate projects, so the results will be the same for both. In most cases, using one of these two ways to migrate the project.json to csproj is the only thing that is needed and no further manual editing of the csproj is necessary. The resulting csproj file will be named per the `name` property in project.json or, lacking that, per containing directory name. 
+
+Visual Studio 2017 will migrate the project automatically when you open either the `xproj` file or the solution file which references `xproj` files. If a solution file is opened, Visual Studio 2017 will automatically migrate all of the projects that are specified in the solution. 
+
+In the command line scenario, you can use the `dotnet migrate` command. It will migrate a project, a solution or a set of folders in that order, depending on which ones were found. 
```

Comment:
Worth mentioning that when you migrate a project, it migrates that project and all its dependencies?